_sass/jekyll-theme-midnight.scss

```diff
@@ -88,7 +88,7 @@ a.anchor {
   }
 
   &::before {
-    content: '🔗';
+    content: '\01F517';
```

Comment:
Maybe add a comment mentioning this character's name, to make the code more self-explanatory. 🙂